src/index.html

```diff
@@ -7,6 +7,382 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a href="#apple" class="nav__link_input is-active">apple</a>
+          </li>
+          <li class="nav__link">
+            <a href="#samsung" class="nav__link_input">samsung</a>
+          </li>
+          <li class="nav__link">
+            <a href="#phones" class="nav__link_input">смартфоны и телефоны</a>
+          </li>
+          <li class="nav__link">
+            <a
+              href="#pc"
+              class="nav__link_input"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__link">
+            <a href="#gajets" class="nav__link_input">гаджеты</a>
+          </li>
+          <li class="nav__link">
+            <a href="#pabs" class="nav__link_input">планшеты</a>
+          </li>
+          <li class="nav__link">
+            <a href="#photo" class="nav__link_input">фото</a>
+          </li>
+          <li class="nav__link">
+            <a href="#video" class="nav__link_input">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div class="catalog__container">
+        <div class="card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="imac-image"
+            class="card__photo"
+          >
+          <div class="card__description">
+            <h2 class="card__name">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h2>
+            <p class="card__code">
+              Код товара: 195434
+            </p>
+          </div>
+          <div class="card__reviews">
+            <div class="card__stars_active">
+              <div class="card__star_active"></div>
+              <div class="card__star_active"></div>
+              <div class="card__star_active"></div>
+              <div class="card__star_active"></div>
+              <div class="card__star_active"></div>
+            </div>
```

Comment:
Don't hardcode it. Use logic written in layotut_stars task. Let the block modifier change the number of active stars (e.g. stars stars--4)